pom.xml

```diff
@@ -10,7 +10,7 @@
 
     <properties>
         <github.url>scm:git:git@github.com:galaxyproject/dockstore-galaxy-interface.git</github.url>
-        <dockstore.version>1.9.0-alpha.1</dockstore.version>
+        <dockstore.version>1.11.2</dockstore.version>
```

Comment:
Shouldn't this be brought up to 1.13.x?

Reply:
Was going to do this in the next PR, this is just getting things working (and scanning) with minimal changes